lib/timespan.py

```diff
@@ -0,0 +1,89 @@
+import re
+import calendar
+from tasks.meta import OBSTimespan, current_session
+
+QUARTERS = {1: 'first',
+            2: 'second',
+            3: 'third',
+            4: 'fourth'}
+
+
+def get_timespan(timespan_id):
+    ts_id, ts_name, ts_description, ts_timespan = parse_timespan(timespan_id)
+    timespan = OBSTimespan(id=ts_id, name=ts_name, description=ts_description, timespan=ts_timespan)
+    return current_session().merge(timespan)
```

Comment:
ph34r with adding objects to the sessions but is the way we're currently doing it